src/Stripe.net/Services/BankAccounts/BankAccountListOptions.cs

```diff
@@ -0,0 +1,8 @@
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class BankAccountListOptions : StripeListOptions
```

Comment:
Why not `StripeBankAccountListOptions`? I guess it's because of the class name discrepancies but it makes it confusing

Reply:
Same answer as for `CountrySpec`: to maintain internal consistency with the other BankAccount classes (entity, service and other options classes). Also something that we should fix separately.

Reply:
I see but it seems better (to me) to keep consistency with the new naming of the `ListOptions` than the parent class itself. Especially as those are the only 2 that are different

Reply:
I disagree, but since we're going to release a new major version anyway, I guess now's as good a time as any to fix all those classes :)

Reply:
sounds good :)